lighthouse-core/runner.js

```diff
@@ -137,15 +141,25 @@ class Runner {
         configSettings: settings,
         categories,
         categoryGroups: runOpts.config.groups || undefined,
-        timing: {total: Date.now() - startTime},
+        timing: {entries: artifacts.Timing || [], total: 0},
```

Comment:
`artifacts.Timing` is a base artifact, so it needs to exist. If the concern is 3.0 fallback, maybe add a note to remove the fallback after 4.0 (or that it's for lhr ≤ 3.0.3 or whatever)?